Sources/AST/Token.swift

```diff
@@ -115,6 +115,7 @@ extension Token {
       case boolean(BooleanLiteral)
       case decimal(DecimalLiteral)
       case string(String)
+      case address(Int)
```

Comment:
A Swift Int is not going to be big enough to hold a 160 bit address, is it? I think a larger data type should go here, and add a test to make sure the whole range of addresses is covered.

Reply:
Almost certainly true. Did not think about that.